phpBB/language/en/common.php

```diff
@@ -169,6 +169,10 @@
 		1	=> '%d character',
 		2	=> '%d characters',
 	),
+	'CHARACTERS_XY'			=> array(
```

Comment:
May be use of short array notation makes sense in new code.

Reply:
Generally yes, but I think especially for language files this can be done in one go. For this file it's the cleaner approach for this small addition.